airtime_mvc/application/common/getid3/extension.cache.dbm.php

```diff
@@ -0,0 +1,251 @@
+<?php
```

Comment:
It can. (https://github.com/JamesHeinrich/getID3) I tried doing it but must've done something wrong and since I needed it for my site I just went ahead and included it to LB. I understand. Keep in mind I'm still new to doing things with git and haven't had much time to dig in, actually my first time contributing so please bear with me.

Reply:
I'll try helping on getting this landed for sure. Would doing and landing the composer change for you help in any way?

I could either do the change and push to you branch (if you have the allow maintainers edit checkbox set on your PR) or I could add the ID3 lib in a seperate PR and you could merge/rebase yours. Let me know what you would prefer.

Reply:
Thanks. Either way would work. Yes, the allow maintainers edit is checked so you can do the change.

Reply:
Before I get to it, please bear in mind that you will need to update your local working copy once any change I do is pushed (ie. by running `git pull`).

We'll probably want to squash a few of your edits when we merge this but I guess we can wait until everything looks good before we squash.